content/sensu-go/6.7/observability-pipeline/observe-schedule/checks.md

```diff
@@ -681,6 +688,97 @@ Value | Description |
 `monthly` | Subdue once per month on the same day, at the same time of day
 `annually` | Subdue once per year on the same day, at the same time of day
 
+#### Repeat and multi-day subdues
+
+Because repeat schedules for subdues are based only on the specified time of day, you may need to configure more than one repeat for multi-day subdues (like weekends).
```

Comment:
```suggestion
Because repeat schedules for subdues are based only on the specified time of day, you may need to configure more than one repeat for multi-day subdues.
```